rust/arrow/src/compute/kernels/cast.rs

```diff
@@ -443,6 +456,13 @@ pub fn cast(array: &ArrayRef, to_type: &DataType) -> Result<ArrayRef> {
             ))),
         },
 
+        // start decimal casts
+        (Int8, Decimal(p, s)) => cast_numeric_to_decimal::<Int8Type>(array, *p, *s),
+        (Int16, Decimal(p, s)) => cast_numeric_to_decimal::<Int16Type>(array, *p, *s),
+        (Int32, Decimal(p, s)) => cast_numeric_to_decimal::<Int32Type>(array, *p, *s),
+        (Int64, Decimal(p, s)) => cast_numeric_to_decimal::<Int64Type>(array, *p, *s),
+        // end numeric casts
```

Comment:
I think this means to me

```suggestion
        // end decimal casts
```

Reply:
Thanks